input-output-hk/plutus
# Cardano node - hello world.

How to test it:
1. (optional?) `test-node $ ./cleanup.sh`
2. `test-node $ ./start-node-01.sh` (in terminal 1)
3. `test-node $ ./start-node-02.sh` (in terminal 2)
4. `tx-inject $ rm -f node-server.sock && ln -s ../test-node/node1/node.sock node-server.sock` (once, or something to that effect in windows - you can also change the path in`config.yaml`)
5. `tx-inject $ ./start-all-servers --without-mock-node` (in terminal 3)

At this point you should start seeing errors on the output in terminal 1.

Pre-submit checklist:
- Branch
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
    - [x] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [x] Useful pull request description
    - [ ] Reviewer requested

Pre-merge checklist:
- [ ] Someone approved it
- [ ] Commits have useful messages
- [ ] Review clarifications made it into the code
- [ ] History is moderately tidy; or going to squash-merge
